src/style.css

```diff
@@ -47,6 +47,12 @@
   text-decoration: underline;
   color: #000;
 }
+.item-source {
```

Comment:
I counted 4 different types of fonts. That's not cool) It's enough to use one primary custom font and one secondary. 
In this case - less is better.

Reply:
Removed extra fonts.